mne/viz/evoked.py

```diff
@@ -345,7 +345,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
                     # plot channels
                     for ch_idx, z in enumerate(z_ord):
                         line_list.append(ax.plot(times, D[ch_idx], picker=3.,
-                                                 zorder=1 + z,
+                                                 zorder=z + 1 if spatial_colors else 1,
```

Comment:
It used to be 1:

https://github.com/mne-tools/mne-python/blob/b5d5194151fa12e839a55495932fdbfe81d73795/mne/viz/evoked.py#L316

But maybe it should be the MPL default. It should definitely be in front of spines though.